src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskAssignByIdControllerTest.java

```diff
@@ -201,323 +147,8 @@ public void assigner_has_grant_type_specific_and_permission_manage_own_can_assig
-        //assigner role : read, manage, own, cancel
-        //assignee role : read, manage, own, cancel
-        TestVariables taskVariables = common.setupWATaskAndRetrieveIds("requests/ccd/wa_case_data.json", "reviewSpecificAccessRequestLegalOps");
-        taskId = taskVariables.getTaskId();
-
-        common.setupLeadJudgeForSpecificAccess(assignerCredentials.getHeaders(), taskVariables.getCaseId(), WA_JURISDICTION);
-
-        initiateTask(taskVariables, Jurisdiction.WA);
-
-        common.setupLeadJudgeForSpecificAccess(assigneeCredentials.getHeaders(), taskVariables.getCaseId(), WA_JURISDICTION);
-        assignTaskAndValidate(taskVariables, getAssigneeId(assigneeCredentials.getHeaders()));
-
-        common.cleanUpTask(taskId);
-    }
-
-    @Test
-    public void assigner_has_grant_type_specific_and_permission_read_manage_own_cancel_can_assign_other_already_assigned_task() {
-        //assigner role : read, manage, own, cancel
-        //assignee role : read, manage, own, cancel
-        TestVariables taskVariables = common.setupWATaskAndRetrieveIds("requests/ccd/wa_case_data.json", "reviewSpecificAccessRequestLegalOps");
-        taskId = taskVariables.getTaskId();
-
-        common.setupLeadJudgeForSpecificAccess(assignerCredentials.getHeaders(), taskVariables.getCaseId(), WA_JURISDICTION);
-
-        initiateTask(taskVariables, Jurisdiction.WA);
-
-        //first assign
-        common.setupLeadJudgeForSpecificAccess(assigneeCredentials.getHeaders(), taskVariables.getCaseId(), WA_JURISDICTION);
-        assignTaskAndValidate(taskVariables, getAssigneeId(assigneeCredentials.getHeaders()));
-
-        //second assign
-        common.setupLeadJudgeForSpecificAccess(secondAssigneeCredentials.getHeaders(), taskVariables.getCaseId(), WA_JURISDICTION);
-        assignTaskAndValidate(taskVariables, getAssigneeId(assigneeCredentials.getHeaders()));
-
-        common.cleanUpTask(taskId);
-    }
-
-    @Test
-    public void assigner_has_grant_type_specific_and_permission_read_manage_execute_cancel_can_assign_task() {
-        //assigner role : read, manage, execute, cancel
-        //assignee role : read, manage, execute, cancel
-        TestVariables taskVariables = common.setupWATaskAndRetrieveIds("requests/ccd/wa_case_data.json", "reviewSpecificAccessRequestLegalOps");
-        taskId = taskVariables.getTaskId();
-
-        common.setupCaseManagerForSpecificAccess(assignerCredentials.getHeaders(), taskVariables.getCaseId(), WA_JURISDICTION, WA_CASE_TYPE);
-
-        initiateTask(taskVariables, Jurisdiction.WA);
-
-        common.setupCaseManagerForSpecificAccess(assigneeCredentials.getHeaders(), taskVariables.getCaseId(), WA_JURISDICTION, WA_CASE_TYPE);
-        assignTaskAndValidate(taskVariables, getAssigneeId(assigneeCredentials.getHeaders()));
-
-
-        common.cleanUpTask(taskId);
-    }
-
-    @Test
-    public void assigner_has_grant_type_specific_and_permission_read_manage_execute_cancel_can_assign_other_already_assigned_task() {
```

Comment:
candidate for IT test? discuss? As long as we have one permission match?

Reply:
same scenario as comment for assigning an already assigned task